source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2386,6 +2386,48 @@ lldb::addr_t SwiftLanguageRuntime::FixupAddress(lldb::addr_t addr,
   return addr;
 }
 
+const swift::reflection::TypeInfo *
+SwiftLanguageRuntime::GetTypeInfo(CompilerType type) {
+  swift::CanType swift_can_type(GetCanonicalSwiftType(type));
+  ConstString mangled_name(type.GetMangledTypeName());
```

Comment:
Does GetMangledTypeName use the debugger mangling? I think with a CanType there should be no difference (CanTypes don't contain Type Aliases) but just to be safe, you might want to write some tests that a global variable of type alias type still works (remote mirrors doesn't support type alias types since there's no alias -> underlying type mapping in the reflection metadata).

Reply:
We got this sorted out: Tuples with resilient elements become one single fixed-size buffer.